src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusMessagePumpOptions.cs

```diff
@@ -152,14 +93,13 @@ public int MaximumUnauthorizedExceptionsBeforeRestart
         /// <summary>
         /// Gets the consumer configurable options model to change the behavior of the tracked Azure Service bus request telemetry.
         /// </summary>
+#pragma warning disable CS0618 // Type or member is obsolete
         public MessageTelemetryOptions Telemetry => Routing.Telemetry;
+#pragma warning restore CS0618 // Type or member is obsolete
 
         /// <summary>
         /// Gets the default consumer-configurable options for Azure Service Bus Queue message pumps.
         /// </summary>
-        internal static AzureServiceBusMessagePumpOptions DefaultOptions => new AzureServiceBusMessagePumpOptions()
-        {
-            TopicSubscription = ServiceBus.TopicSubscription.None
-        };
+        internal static AzureServiceBusMessagePumpOptions DefaultOptions => new();
```

Comment:
It's a detail, but I'm just wondering if an `internal static readonly` member isn't better here then an `internal static` property ?
The semantics are somewhat different I guess, where an `internal static readonly` is only initialized once, and is only initialized when the member is accessed for the first time.
With a property implementation like it is now, I think we'll be initializing and returning a new instance every time we access the property.

Reply:
Why, I would assume that this is just a (readonly) instance that just contains the default settings ?  Why would `DefaultOptions` be mutable ?

Reply:
Because it is passed as a 'default/initial' value for the `Action<...> configureOptions` parameter. The instance can get mutated there by the user.